velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -241,12 +243,13 @@ void WindowFuzzer::go() {
       sortingKeysAndOrders =
           generateSortingKeysAndOrders("s", argNames, argTypes);
     }
-    auto partitionKeys = generateSortingKeys("p", argNames, argTypes);
-    auto frameClause = generateFrameClause();
-    auto input = generateInputDataWithRowNumber(argNames, argTypes, signature);
-    // If the function is order-dependent, sort all input rows by row_number
-    // additionally.
-    if (requireSortedInput) {
+    const auto partitionKeys = generateSortingKeys("p", argNames, argTypes);
+    const auto [frameClause, isRowsFrame] = generateFrameClause();
+    const auto input =
+        generateInputDataWithRowNumber(argNames, argTypes, signature);
+    // If the function is order-dependent or uses "rows" frame, sort all input
+    // rows by row_number additionally.
+    if (requireSortedInput || isRowsFrame) {
```

Comment:
@kagamiori : I agree that its better to use row_number for sorting for all rows frame to obtain deterministic results. However, your PR description suggested you only needed for CURRENT ROW. Please can you fix that.